core/src/main/java/org/stellar/anchor/config/ClientsConfig.java

```diff
@@ -1,17 +1,21 @@
 package org.stellar.anchor.config;
 
+import com.google.gson.annotations.SerializedName;
 import java.util.List;
-import org.stellar.anchor.client.CustodialClientConfig;
-import org.stellar.anchor.client.NonCustodialClientConfig;
+import org.stellar.anchor.client.TempClient;
 
 public interface ClientsConfig {
```

Comment:
Further reviewed the code, I think we don't need to use `interface`. If the implementation complexity can be reduced, it would be ok to use `class PropertyClientsConfig` directly and remove the `ClientsConfig`.

Reply:
let me double check on that

Reply:
Remove it will cause the circular dependency platform.main and core.main

Reply:
The `ClientsConfig` and `DefaultClientService` are not used in `core`, it is better to move them to `platform` if possible.

Reply:
I would prefer to do it in a separate PR as tho these also applies to several other files.

Reply:
That's no problem.